api/data/it/PD.tsx

```diff
@@ -0,0 +1,431 @@
+import { Box, Button, Card, CardActions, CardContent, Container, Grid, Link, Stack, Typography } from "@mui/material";
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import CodeIcon from '@mui/icons-material/Code';
+
+export const Cards1 = () => {
+    const cards1Data = [
+        {
+            title: "1. Termini e Condizioni di adesione e uso",
+            buttons: [
+                {
+                    label: "Leggi i Termini e Condizioni",
+                    color: "primary",
+                    link: "https://docs.pagopa.it/documento-1-termini-condizioni-di-adesione-e-uso/",
+                    icon: <ArrowForwardIcon sx={{ color: "#0073E6" }} />,
+                },
+            ],
+        },
+        {
+            title: "2. Atto di Nomina a Responsabile Trattamento Dati Personali",
+            buttons: [
+                {
+                    label: "Leggi l’Atto di Nomina",
+                    color: "primary",
+                    link: "https://docs.pagopa.it/doc.2-atto-di-nomina-a-responsabile-trattamento-da/",
+                    icon: <ArrowForwardIcon sx={{ color: "#0073E6" }} />,
+                },
+            ],
+        },
+        {
+            title: "3. Documentazione tecnica",
+            buttons: [
+                {
+                    label: "Vai al documento",
+                    color: "primary",
+                    link: "#",
+                    icon: <ArrowForwardIcon sx={{ color: "#0073E6" }} />,
+                },
+                {
+                    label: "Leggi il manuale operativo",
+                    color: "primary",
+                    link: "#",
```

Comment:
@salvatorediocaro aggiungi il link al manuale operativo
https://docs.pagopa.it/manuale-operativo/